### PR TITLE
[No Merge] Feature/new end game sequence

### PR DESCRIPTION
**Pull request by sc0ttr0berts0n**
No merge until @BeanSparrow gives the okay.

### CODE REVIEW

**Comment by sc0ttr0berts0n** [comment]
I like this, but I wouldn't mind seeing an option where:

- The blue walls become grey too.
- The entire background except for the robot / blood fades to black, but not fully.

File: js/game.js [hunk 1]
```diff
@@ -74,6 +77,14 @@ class Game {
             this.flaks.forEach(flak => flak.update());
             this.flaks = this.flaks.filter(flak => !flak.isDead);
         }
+        this.endGameUpdate();
+    }
+    endGameUpdate() {
+        if (!this.player.alive && this.deadTime < 5) {
+            this.deadTime++;
+            this.app.stage.filters = [this.colorMatrix];
```
Comment (sc0ttr0berts0n) [inline js/game.js]: why are you passing this as a symbol? `[this.colorMatrix]` @BeanSparrow

File: js/game.js [hunk 2]
```diff
@@ -74,6 +77,14 @@ class Game {
...
+            this.colorMatrix.greyscale(0.9);
```
Comment (sc0ttr0berts0n) [inline js/game.js]: maybe `0.6` here.

File: js/game.js (hunk 1 above)
Comment (BeanSparrow) [reply js/game.js]: it's because a PIXI.ColorMatrixFilter is a literal matrix

**Comment by sc0ttr0berts0n** [comment]
@BeanSparrow I have plans to do some codebase tweaks that could ruin the compatibility here. Is this still on your radar to merge in?

**Comment by sc0ttr0berts0n** [comment]
Won't merge. Incompatible now.

**Pull request closed without merging**